README.md

```diff
@@ -1,75 +1,29 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> Description the project.
+# Portfolio Mobile version
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
```

Comment:
- Please, kindly add the "Getting Started" section and some instructions on how users can have your project run locally on their machine.. e.g how to set up, clone the repository, and prerequisite.